p2p/dandelion/src/tests/router.rs

```diff
@@ -6,6 +6,59 @@ use crate::{DandelionConfig, DandelionRouteReq, DandelionRouter, Graph, TxState}
 
 use super::*;
 
+/// make sure the number of stemm peers is correct.
+#[tokio::test]
+async fn number_stems_correct() {
+    let mut config = DandelionConfig {
+        time_between_hop: Duration::from_millis(175),
+        epoch_duration: Duration::from_secs(60_000),
+        fluff_probability: 0.0, // we want to be in stem state
+        graph: Graph::FourRegular,
+    };
+
+    let (broadcast_svc, _broadcast_rx) = mock_broadcast_svc();
+    let (outbound_peer_svc, _outbound_rx) = mock_discover_svc();
+
+    let mut router = DandelionRouter::new(broadcast_svc, outbound_peer_svc, config.clone());
```

Comment:
https://github.com/Cuprate/cuprate/actions/runs/8944812790/job/24572545093?pr=111
```suggestion
    let mut router = DandelionRouter::new(broadcast_svc, outbound_peer_svc, config);
```